### PR TITLE
更新 open graph資訊

### LINKED ISSUE #119: Open graph setting update
Update to new URL https://worktime.goodjob.life and update og-image
- [ ] Update `og-` attribute
- [ ] Update FB share links

### PR DESCRIPTION
**Pull request by barry800414**
to fix #119

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
     <meta name="author" content="">
     <meta property="og:title" content="工時透明化運動">
     <meta property="og:type" content="website">
-    <meta property="og:image" content="https://goodjoblife.github.io/WorkTimeSurvey/img/intro-bg.jpg">
-    <meta property="og:url" content="https://goodjoblife.github.io/WorkTimeSurvey/">
+    <meta property="og:image" content="https://worktime.goodjob.life/img/og-image_1200-630.png">
+    <meta property="og:url" content="https://worktime.goodjob.life/show.html">
```
Comment (mark86092) [inline index.html]: wrong url

**Comment by mark86092** [comment]
Please remove `intro-bg.jpg`, it will not be used any more

File: about_us.html [hunk 2]
```diff
@@ -12,8 +12,8 @@
     <meta property="og:title" content="工時透明化運動 - 關於我們">
     <meta property="og:type" content="website">
     <meta property="og:image" content="https://goodjoblife.github.io/WorkTimeSurvey/img/intro-bg.jpg">
```
Comment (qcl) [inline about_us.html]: This `og:image` should be update

**Comment by qcl** [comment]
I've done my review.

File: index.html (hunk 1 above)
Comment (barry800414) [inline index.html]: 哪個地方錯了?

File: about_us.html (hunk 2 above)
Comment (barry800414) [inline about_us.html]: 好奇怪我明明有更新QQ
來處理

**Comment by mark86092** [comment]
OK, I reviewed it

**Comment by qcl** [comment]
👍  :shipit:

**Pull request merged**